lib/BuildSystem/SwiftTools.cpp

```diff
@@ -220,6 +223,11 @@ class SwiftCompilerShellCommand : public ExternalCommand {
     if (isLibrary) {
       result.push_back("-parse-as-library");
     }
+    if (enableWholeModuleOptimization) {
+      result.push_back("-whole-module-optimization");
+      result.push_back("-num-threads");
+      result.push_back("1");
```

Comment:
We don't have a good answer currently, for now I think the best thing is to make it a separate option and handle both cases of parallel or non-parallel WMO. Then clients can pick if they want to hard code a threads value even if llbuild itself isn't scheduling it "properly"